haskell/test/Test/Ligo/TreasuryDAO.hs

```diff
@@ -100,8 +103,8 @@ flushTokenTransfer = uncapsNettest $ withFrozenCallStack $ do
   withSender (AddressResolved owner2) $
     call dao (Call @"Freeze") (#amount .! 20)
 
-  -- Advance one voting period (which is 1).
-  advanceTime (sec 1.5)
+  -- Advance one voting period (which is 10).
+  advanceTime (sec 20)
```

Comment:
This seems incorrect (at least the comment), I suppose the idea here was to advance a little more than a period (but not two).

Reply:
>a little more than a period (but not two).

But since periods start with proposing, if only little more then a period was advanced, it would end up in a voting period, right? In that case we would not be able to raise the proposal, which appear to be the next step in this test.

I have updated it with the comment amended.